ale_linters/typescript/xo.vim

```diff
@@ -0,0 +1,23 @@
+call ale#Set('typescript_xo_executable', 'xo')
+call ale#Set('typescript_xo_use_global', get(g:, 'ale_use_global_executables', 0))
+call ale#Set('typescript_xo_options', '')
+
+function! ale_linters#typescript#xo#GetExecutable(buffer) abort
+    return ale#node#FindExecutable(a:buffer, 'typescript_xo', [
+    \   'node_modules/.bin/xo',
+    \])
+endfunction
+
+function! ale_linters#typescript#xo#GetCommand(buffer) abort
+    return ale#Escape(ale_linters#typescript#xo#GetExecutable(a:buffer))
+    \   . ale#Pad(ale#Var(a:buffer, 'typescript_xo_options'))
+    \   . ale#Pad('--reporter unix --stdin --stdin-filename %s')
```

Comment:
```suggestion
    \   . ' --reporter unix --stdin --stdin-filename %s'
```

You can change this back. `ale#Pad()` is just useful for adding a space before options if they aren't an empty string.